packages/core/content-type-builder/admin/src/pages/App/TempTP.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
```

Comment:
Do you still need the theme provider ?

I think you can just comment the LV as follows and remove the theme provider, it will also helps us make sure we have fully migrated all the components to parts.

```
export default () => 'TODO LV';
```

Reply:
I will remove it when I'm done, I'm more comfortable with this approach when it comes to big features like this one